docs/src/main/paradox/client.md

```diff
@@ -179,3 +179,15 @@ java.util.logging.ConsoleHandler.formatter=java.util.logging.SimpleFormatter
 ```
 
 Run with `-Djava.util.logging.config.file=/path/to/grpc-debug-logging.properties`.
+
+## Configuration
+
+You can configure where your .proto files are located like this:
+
+sbt
+:   @@snip[build.sbt]($root$/../plugin-tester-java/build.sbt) { #protoSources }
+
+TODO gradle, maven
+
+TODO describe settings for code generation: scala/java, client/server/both
+
```

Comment:
I started on these as well in the Both-PR. I'm thinking it makes more sense with a separate section for each build tool than mashing them all together?

Reply:
Yeah it's tricky having so many axes: client/server, java/scala, maven/gradle/sbt, I'm undecided. Perhaps we should keep the 'getting started' above 'shared' between build tools, but do separate 'configuration' sections for each one?

Reply:
Yeah, that was exactly my thinking!

Reply:
+1 for separation by build tool here